app.js

```diff
@@ -1,13 +1,27 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
+const { getEndpoints } = require("./controllers/api.controllers");
+
 const app = express();
 
 // TODO: when receiving a json body from the client, e.g. POST express.json()
 
 app.get("/api/topics", getTopics);
 
+app.get("/api", getEndpoints);
+
+// ERROR HANDLING
+app.use((err, req, res, next) => {
+	// handle error if endpoint.json file is missing. Cannot test for this as the filename is not given as part of endpoint, but manually confirmed in development with malformed path.
```

Comment:
remove all comments and console.logs before pushing from now on

Reply:
removed comments and consoles.